coord/src/rmr/command.c

```diff
@@ -245,56 +116,20 @@ static void extendCommandList(MRCommand *cmd, size_t toAdd) {
   cmd->lens = rm_realloc(cmd->lens, sizeof(*cmd->lens) * cmd->num);
 }
 
-void MRCommand_AppendStringsArgs(MRCommand *cmd, int num, char **args) {
-  if (num <= 0) return;
-  int oldNum = cmd->num;
-  extendCommandList(cmd, num);
-
-  for (int i = oldNum; i < cmd->num; i++) {
-    assignCstr(cmd, i, args[i]);
-  }
-}
-
-void MRCommand_AppendArgsAtPos(MRCommand *cmd, int pos, int num, ...) {
-  if (num <= 0) return;
+void MRCommand_Insert(MRCommand *cmd, int pos, const char *s, size_t n) {
   int oldNum = cmd->num;
-  extendCommandList(cmd, num);
+  extendCommandList(cmd, 1);
```

Comment:
We always use it with 1?

Reply:
Maybe we could have keep this function and add another function that uses it with single element? (not ciritical..)

Reply:
If someone wants to have a function to append/insert multiple arguments, he can implement it... For better coverage and as we don't need it at the moment, 1 is enough